CMake/FindBlueZ.cmake

```diff
@@ -6,7 +6,7 @@
 #
 # vim: expandtab sw=4 ts=4 sts=4:
 
-include(FindPkgConfig)
+find_package(PkgConfig QUIET)
```

Comment:
I haven't looked at many of our cmake files but should we be using `REQUIRED` here since we use it on the next line?

Reply:
I am not sure, none of the other cmake files in dolphin-emu do it.
```
$ find . -type f -iname '*cmake*' | xargs grep PkgConfig
./CMake/FindSDL2.cmake:find_package(PkgConfig QUIET)
./CMake/CheckLib.cmake:find_package(PkgConfig)
./CMake/FindFFmpeg.cmake:    find_package(PkgConfig QUIET)
./CMake/FindEGL.cmake:find_package(PkgConfig)
./CMake/FindBlueZ.cmake:find_package(PkgConfig QUIET)
./CMake/FindLIBEVDEV.cmake:find_package(PkgConfig)
./CMake/FindPulseAudio.cmake:find_package(PkgConfig QUIET)
./CMake/FindSYSTEMD.cmake:find_package(PkgConfig QUIET
```
And neither does upstream cmake.
```
$ cd /usr/share/cmake-3.18/
$ grep -r 'find_package(PkgConfig'
Help/manual/cmake-developer.7.rst:  find_package(PkgConfig)
Modules/FindLibXslt.cmake:find_package(PkgConfig QUIET)
Modules/FindCurses.cmake:find_package(PkgConfig QUIET)
Modules/FindOpenSSL.cmake:  find_package(PkgConfig QUIET)
Modules/FindFontconfig.cmake:find_package(PkgConfig QUIET)
Modules/FindImageMagick.cmake:find_package(PkgConfig QUIET)
Modules/FindGSL.cmake:  find_package(PkgConfig)
Modules/FindLibXml2.cmake:find_package(PkgConfig QUIET)
Modules/FindLibinput.cmake:find_package(PkgConfig QUIET)
Modules/FindGnuTLS.cmake:  find_package(PkgConfig QUIET)
Modules/FindBLAS.cmake:  find_package(PkgConfig)
Modules/FindEXPAT.cmake:find_package(PkgConfig QUIET)
Modules/FindCURL.cmake:find_package(PkgConfig QUIET)
```